packages/components/src/focal-point-picker/index.tsx

```diff
@@ -237,15 +236,6 @@ export function FocalPointPicker( {
 	const instanceId = useInstanceId( FocalPointPicker );
 	const id = `inspector-focal-point-picker-control-${ instanceId }`;
 
-	useUpdateEffect( () => {
-		setShowGridOverlay( true );
-		const timeout = window.setTimeout( () => {
-			setShowGridOverlay( false );
-		}, GRID_OVERLAY_TIMEOUT );
```

Comment:
I've omitted this logic from the update. I'm happy to restore it, but I've not noticed a big difference.

Reply:
In general I'm a fan of avoiding effects. The difference this makes is that using the inputs to change the focal point no longer shows the grid overlay.

Possibly there’s a way to rework how the `Grid` visibility changes so that it show/hides without using state and avoid the effect but I'm not sure.

I don’t have much of an opinion on whether it’s okay to omit showing the grid for changes from the inputs.